src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -249,7 +249,8 @@
 
       <Crossgen2SupportedRids Include="@(Net60Crossgen2SupportedRids)" />
 
-      <ILCompilerSupportedRids Include="linux-musl-x64;linux-x64;linux-arm64;win-x64;win-arm64" />
+      <!-- Match the ILCompilerRIDs.props at https://github.com/dotnet/runtime/blob/main/src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/ILCompilerRIDs.props -->
+      <ILCompilerSupportedRids Include="linux-arm;linux-musl-arm64;linux-arm64;linux-musl-x64;linux-x64;osx-x64;win-arm64;win-x64" />
```

Comment:
Is there any harm in listing all possible RIDs here, e.g. including osx-arm64 that we do not build at the moment but that is likely going to be added soon?

Looks good to me otherwise.

Reply:
Currently we are using local package (osx-arm64 8.0.0-dev) for port work, so the bundled RID check does not block.

Generally, I think it scales well if we avoid lists of RIDs or platforms where possible. e.g. UX-wise, it should be alright to get an I/O, network error for unsupported platform, without explicitly maintaining a list.